baryshev_code_HW7.py

```diff
@@ -0,0 +1,80 @@
+from pydrive.auth import GoogleAuth
+from pydrive.drive import GoogleDrive
+
+# Creates local webserver and auto handles authentication
+gauth = GoogleAuth()
+gauth.LocalWebserverAuth()
+
+drive = GoogleDrive(gauth)
+
+"""
+Function GetGDriveContent
+Input:
+    drive - GoogleDrive to be accessed
+Output:
+    list_of_content - list of files and folders in the GoogleDrive root (list)
+"""
+def GetGDriveContent(drive):
+    # The line below auto-iterates through all files that matches this query
+    file_list = drive.ListFile({'q':
+        "'root' in parents and trashed=false"}).GetList()
+    list_of_content = []
+    for file1 in file_list:
+        list_of_content.append(file1['title'])
+    return list_of_content
+
+
+"""
+Function CreateFileOnGDrive creates a file with name 'filename'
+in GDrive root directory
+Input:
+    drive - GoogleDrive to be accessed
+    filename - name of the file to be created (string)
+    content - content to be inserted in the file (string)
+Output:
+    None
+"""
+def CreateFileInGDriveRoot(drive,filename,content):
+    GDriveContent = GetGDriveContent(drive)
+    if filename in GDriveContent:
+        print 'A file with name', filename, 'arleady exists in GDrive'
+    else:
+        file1 = drive.CreateFile({'title': filename})
+        file1.SetContentString(content)
+        file1.Upload()
+
+
+def GetFolderID(drive,foldername):
```

Comment:
Unlike the above function that creates a file in the GDrive root, creating a file in a particular folder isn't straightforward. As I understood documentation, it requires to have the id of the folder where one wants to insert a file. Maybe there is a simpler way to do this.